kolibri/plugins/coach/assets/src/views/reports/ReportsQuizLearnerListPage.vue

```diff
@@ -42,9 +42,13 @@
               </KLabeledIcon>
             </td>
             <td>
-              <StatusSimple :status="tableRow.statusObj.status" />
+              <StatusSimple 
+                :status="tableRow.statusObj.status" 
+                :answeredQuestionsCount="numAnswered(tableRow.statusObj)" 
+                :totalQuestionsCount="exam.question_count" 
+              />
```

Comment:
The `StatusSimple` is actually just for "learner" statuses. All strings describe something of the form "how many learners are ____". (e.g. 3 of 5 learners have started, 1 learner needs help). In this new variation we're talking about questions, not learners.

There actually already are a couple strings that could be helpful:

https://github.com/learningequality/kolibri/blob/47218d290259abc8f9710943f06d599c00674ce8/kolibri/plugins/coach/assets/src/views/common/status/statusStrings.js#L150-L161

So then we can do something like 

```html
<StatusSimple
  v-if="tableRow.numAnswered !== null && exam.question_count !== null"
  :status="tableRow.statusObj.status"
/>
<KLabeledIcon
  v-else
  icon="inProgress"
  :text="translations.question.completed('ratioShort', {count: tableRow.numAnswered, total: exam.question_count})"
/>
```

where `translations` is available from `statusStringsMixin`.

Reply:
@indirectlylit I tried applying these changes - but it looks like `statusStringsMixin` also brings along with it a bunch of props (count, verbosity, total) - which are required - which are not passed to `ReportsQuizLearnerListPage` - so I'm getting errors for those props not being provided to the component in which I'm using that mixin.

Also - that `ratioShort` string would be `X of Y completed` - which I don't think provides the meaningful context that `X of Y answered` does in implying that the X and Y are referring to a number of questions. 

In any case - should I define that string directly on the component for simplicity's sake?

I have implemented it as follows now - and the one issue I'm having is that the icon is not the light-blue color but is instead black.

```html
<StatusSimple
  v-if="tableRow.statusObj.status !== STATUSES.started"
  :status="tableRow.statusObj.status"
/>
<KLabeledIcon v-else>
  <KIcon slot="icon" inProgress />
  {{ $tr('questionsCompletedRatioLabel', {count: tableRow.statusObj.num_answered, total: exam.question_count}) }}
</KLabeledIcon>
```

Reply:
> In any case - should I define that string directly on the component for simplicity's sake?

I think you're right: better to define a new string specifically for this case

> the icon is not the light-blue color but is instead black unless I pass the hex code directly to the KIcon.

Rather than a hex code, you'll want to use the color token defined here: https://github.com/learningequality/kolibri/blob/develop/kolibri/core/assets/src/styles/theme.js#L40

To use this, add the `themeMixin` and then reference `$themeTokens.progress`